### PR TITLE
Fix comments for PredefinedAcl

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/432)

### CODE REVIEW

File: Google.PowerShell/Provider/GoogleCloudStorageProvider.cs [hunk 1]
```diff
@@ -134,23 +134,37 @@ public class NewGcsBucketDynamicParameters
 
             /// <summary>
             /// <para type="description">
-            /// Default ACL for the bucket. e.g. "publicRead", "private", etc.
+            /// Default ACL for the bucket.
```
Comment (iantalarico) [inline Google.PowerShell/Provider/GoogleCloudStorageProvider.cs]: Not required but I find both of these are a little hard to parse. It would be nice if enum values and "*** permission" were quoted.

**Comment by quoctruong** [comment]
Thanks!
